js/index.js

```diff
@@ -0,0 +1,49 @@
+
+$('#buttons-container').on('click', 'span', printToScreen);
+$('#equals').on('click', calculate);
+$('#clear').on('click', clearScreen);
+
+let entry = [];
+
+function printToScreen(event) {
+  const buttonPushed = $(event.target).text();
+  if (buttonPushed !== 'C' && buttonPushed !== '=') {
+    entry.push(buttonPushed);
+    $('#screen').text(entry.join(''));
+  }
+}
+
+function calculate(event) {
+  const entryString = cleanEntryString();
+  if (checkForValidExpression(entry)) {
+    const result = eval(entryString);
+    $('#screen').text(result);
+    entry = [result];
+  } else {
+    $('#screen').text("Error");
+    entry = [];
+  }
+}
+
+function cleanEntryString() {
+  const entryString = entry.join('').replace('÷', '/').replace('x', '*');
+  return entryString;
+}
+
+function checkForValidExpression(entry) {
+  const entryString = cleanEntryString();
+  //check to see whether there are 2 operands in a row
+  let patternMet = !(/[\+\-\/\*]{2}/).test(entryString);
+  //check to see whether there's division by 0
+  let divideByZero = /(\/0)/.test(entryString);
+  if (patternMet && !divideByZero) {
```

Comment:
Calculating the values and then using them is a good pattern. It allows for cleaner code.